specification/k3po.control/SPEC.md

```diff
@@ -0,0 +1,195 @@
+# K3po Control Protocol
+
+## Abstract
+
+This document defines the K3po control protocol used to communicate with a remote K3po server.
```

Comment:
I'm not sure what do you mean by this. Client sends commands and the server/driver respond with events. This is already there, under 'The Protocol'.